source/utilities.cc

```diff
@@ -211,6 +211,57 @@ namespace WorldBuilder
     }
 
 
+    std::vector<Point<2> >
+    get_bounding_box (const Point<2> &point,
+                      const std::vector<Point<2> > &point_list,
+                      std::size_t original_number_of_coordinates,
+                      const double buffer_around_fault)
+    {
+      std::vector<double> x_list(original_number_of_coordinates,0.0);
+      std::vector<double> y_list(original_number_of_coordinates,0.0);
+
+      for (size_t j=0; j<original_number_of_coordinates; ++j)
+        {
+          x_list[j] = point_list[j][0];
+          y_list[j] = point_list[j][1];
+        }
+
+      // Corner coordinates of the bounding polygon
+      std::vector<Point<2> > bounding_box (4, point);
```

Comment:
What is the meaning of `point` here? You will overwrite all elements of this array of four elements later on, so the value of `point` is unimportant. I don't think you actually need this function argument.

Reply:
Yes, the value of the point is not important. It was just to create the correct data type for bounding_box. Removed it now, thanks!